### PR TITLE
Leaderboard: final touches

### PR DESCRIPTION
**Pull request by ntzwilly**
# Final touches
## Description
 Improve the look and feel of the application by adding the styles

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,48 @@
...
+    1. git https://github.com/ntzwilly/leaderboard.git
+    2. cd leaderboard
+
+### Install
+
+    npm install
+
+### Usage
+
+    npm start
```
Comment (arslanbisharat) [inline README.md:37]: - [ ] Your README is looking professional but it must contain all that's needed for running the app with the API(s) it consumes. It should have instructions about where and how to generate any necessary keys, credentials, how to set, store, and generate them(if necessary).

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 1 above)
Comment (ntzwilly) [reply README.md:37]: Hi, thank you for the review. Apology, I don't understand it. Could you please be more specific in your feedback? I think this is not part of the project's requirements. The key is unique and was generated by the API when I created the game. Also, the application doesn't require any credentials.

**Review by bellom — APPROVED** [review]
## Hi @ntzwilly  👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**